### DISCUSSION TITLE
How to disble password reset throttle?

### DISCUSSION THREAD

**Discussion by realtebo** [Laravel-Backpack/community-forum#749] [Q&A (Help)] [answered]
normally,  in a laravel app, it's enough to disable `'throttle' => \Illuminate\Routing\Middleware\ThrottleRequests::class,` row in `kernel.php`

In backpack, it uses a custom throttling middleware and we cannot disable it. We need to allow [dev only, of course] infinite retry

**Comment by pxpm** [comment, marked as answer]
We don't have a built in way to specifically disable it. but you can use overly big limits in https://github.com/Laravel-Backpack/CRUD/blob/4c7030524540d34ca32db367a48ae03d34c14505/src/config/backpack/base.php#L98 

If you want to make those configs `environment dependent`, you can can just use `'password_recovery_throttle_access' => env('PASSWORD_RECOVERY_THROTTLE_ACCESS', '3,10')`. 

It's not a good practice to "comment code" to work only on local. That's bound for disaster if you forget and commit that to prod. Either have a boolean setting to specifically enable/disable something, or use stupid big limits while testing, without affecting other environments. 

Let me know if that helps.

Cheers

**Reply by realtebo** [reply]
Yes. tweaking 2 configs, it's all we need

    password_recovery_throttle_notifications

and

    password_recovery_throttle_access
